constants/constants.js

```diff
@@ -1 +1,4 @@
-export const TOTALS_STUDY = 'Totals'
+export const constants = {
+  TOTALS_STUDY: 'Totals',
+  OMIT_ACCESS: ['files', 'combined'],
+}
```

Comment:
🎱 could rename this file to `contants/index.js`. Just thinking that it's at the root of the project, so it may help in the future to be able to break constants out into files for specific concerns `constants/foo.js`, `constants/bar.js` and then this file could looks something like
```suggestion
import { fooConstants } from './foo'
import { barConstants } from './bar'

export const constants = {
  TOTALS_STUDY: 'Totals',
  fooConstants,
  barConstants,
}
```

In the short term that would also shorten [line 4](https://github.com/AMP-SCZ/dpdash/pull/147/files#diff-f21caf3f3e5b43fb807227c3256bd6bd1bb353df6834df0813e50e7d2e2a74c6R4) in the next file 🙃 
```diff
- import { constants } from '../../constants/constants'
+ import { constants } from '../../constants'
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/147/commits/04f13bd1ffa2be4dabbec2ecb9bb841fb2abafa3

Reply:
I think resolved in 7fbee241b1d3b4d1368f3807afdf2d6ddc9cbc0a 😉